### ISSUE TITLE
Meteorite spawn options

### ISSUE THREAD

**Issue by generrosity** [AppliedEnergistics/Applied-Energistics-2#7243]
### Describe the feature

The way the documentation is worded suggests that the mood knows when it is attempting to place an underground or surface meteor.

Some previous mods had have the ability to choose to restrict to one of the other, or to set a max-y-level

Looking for this to have a more vanilla+ feel to world generation with mods based on exploration and mods interaction (compass)

### Reasons why it should be considered

Mod pack options, more server owner options

**Comment by Technici4n** [comment]
What do you mean exactly? You'd want an option to prevent meteorites from being placed above some y level?

**Comment by generrosity** [comment]
Yes - that is one way to achieve this

**Comment by Inaaca** [comment]
Yes, the main issue I have with this mod is how frequently the meteorites destroy the natural environment.  They generate far too excessively imo, even in clusters of 2 or more at a time.  A way to reduce (or eliminate entirely) the frequency of them generating on the surface is a needed option.  

The config file used to have this and allowed some control over meteorite generation rates.  Why it was removed I have no idea.

**Comment by Ezoteric** [comment]
I would like to have the following options.

- Spawn rate setting
- Spawn Dimension setting

They spawn way too much and i dont want them to spawn in the overworld.

Thnx in advance

**Comment by Ezoteric** [comment]
> > I would like to have the following options.
> > ```
> > * Spawn rate setting
> > 
> > * Spawn Dimension setting
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > They spawn way too much and i dont want them to spawn in the overworld.
> > Thnx in advance
> 
> This is handled using Vanilla datapacks and there's not going to be (nor can there be) an additional config setting for this.

Thnx m8, yeah sorryi havent been playing for years, is there any datapack guide for those settings or are you willing to show/give me one ?

Thnx in advance

**Comment by Ezoteric** [comment]
> > Thnx m8, yeah sorry I havent been playing for years, is there any datapack guide for those settings or are you willing to show/give me one ?
> 
> No sadly not. Not having a guide for those settings is definitely a shortcoming we have and we have to do something about that. :-( What you _can_ do is extract the AE2 jar and grab the files from data/ae2/worldgen/structure* and put those into a datapack.
> 
> A note on dimensions: You cannot control which dimensions spawn structures. What you can do however is control this via biomes. We use a biome tag to control this (` "biomes": "#ae2:has_meteorites",`) .
> 
> For spawn rate, check the structure_set file.

Thnx m8, thats exactly why i think a config file is for the best as it speaks for it self.
Anyway, i did the following;
copy ae2\worldgen\structure_set\meteorite.json and changed the following:

{
  "placement": {
    "type": "minecraft:random_spread",
    "salt": 124895654,
    "separation": 32,   <-- changed
    "spacing": 196    <-- changed
  },
  "structures": [
    {
      "structure": "ae2:meteorite",
      "weight": 2   <-- changed
    }
  ]
}


I asume weight 2 means 50%, weight 3 means 33%, etc. like most mods? as default was 1
Just testing this out so hope it works :)
